### PR TITLE
add websocketPort to public server list

### PR DESCRIPTION
**Pull request by tooomm**
See https://github.com/Cockatrice/Cockatrice/pull/3545

https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/handle_public_servers.cpp#L74

<br>

I also deprecated dr4ft server as it has been down for quite a while and added some cleanup.

TODO with merge: remove from wiki as well: https://github.com/Cockatrice/Cockatrice/wiki/Public-Servers

### CODE REVIEW

**Comment by tooomm** [comment]
@skwerlman let us know when you updated your server and websockets are supported!

**Comment by skwerlman** [comment]
I now accept SSL websocket connections at mtg.tetrarch.co:443

File: public-servers.json [hunk 1]
```diff
@@ -36,12 +35,14 @@
...
 			"host": "server.cockatrice.us",
 			"port": 4747,
+			"websocketPort": 4748,
 			"authRequired": true,
 			"regRequired": true
 		},
 		{
 			"id": 4,
 			"name": "Woogerworks",
+			"site": https://woogerworks.com/,
```
Comment (tooomm) [inline public-servers.json]: Any clue why the URL is highlighted in red? Is it only me?

Comment (skwerlman) [reply public-servers.json]: its not in quotes

**Comment by tooomm** [comment]
@ZeldaZach I think we are ok to merge here and test with the beta.4 client?

@basicer any objections?

**Pull request merged**